pennylane/gradients/spsa_gradient.py

```diff
@@ -125,16 +126,18 @@ def spsa_grad(
               A valid sampling method can, but does not have to, take this counter into
               account. In any case, ``sampler`` has to accept this third argument.
 
-            - The keyword argument ``seed``, expected to be ``None`` or an ``int``.
-              This argument should be passed to some method that seeds any randomness used in
-              the sampler.
+            - The keyword argument ``rng``, expected to be a NumPy pseudo-random number generator,
```

Comment:
Mention of `rng` being a keyword argument

Reply:
Technically, in `spsa_grad` the `rng` is passed as a keyword argument. And in the `_rademacher_sampler` it is also defined as a _required_ keyword argument. So the `sampler` can accept `rng` as anything that is not a "positional only" argument. I made this more precise in the docs and also added a test for this. https://github.com/PennyLaneAI/pennylane/pull/4165/commits/2eda609429086619842e12da08ccf3a14c68a647

Reply:
Great, thanks for the thorough clarification :)